frontend-implicaction/src/app/forums/components/custom-table-with-search-bar/custom-table-with-search-bar.component.ts

```diff
@@ -131,7 +131,7 @@ export class CustomTableWithSearchBarComponent
   }
 
   protected innerPaginate(): void {
-    if (this.title === 'Forums') {
+    if (this.tableType === 'Forums') {
       this.groupService
         .getAllGroups(this.pageable)
         .pipe(finalize(() => (this.isLoading = false)))
```

Comment:
```suggestion
        .pipe(finalize(() => this.isLoading = false))
```
Les parenthèses n'apportent rien ici => 🗑️